docs/sphinx/source/modelchain.rst

```diff
@@ -232,57 +251,59 @@ plane of array irradiance. The
 assigns default values for temperature (20 C)
```

Comment:
maybe worth removing this sentence now so we don't forget when addressing https://github.com/pvlib/pvlib-python/issues/1113